src/bp/core/routers/admin/versioning.ts

```diff
@@ -74,6 +59,9 @@ export class VersioningRouter extends CustomRouter {
           await this.extractArchiveFromRequest(req, tmpDir.name)
           const newBotIds = await this.ghost.forceUpdate(tmpDir.name)
 
+          this.logger.info(`Unmounting bots: ${beforeBotIds.join(', ')}`)
```

Comment:
Didn't know we could update multiple bots

Reply:
When you pull/push, you replace the entire environment. If you have 5 bots ,then you push 10, we need to unmount the previous ones and mount the new / changed ones